hw/ip/otbn/dv/otbnsim/sim/dmem.py

```diff
@@ -240,12 +243,29 @@ def _commit_store(self, item: TraceDmemStore) -> None:
         # And write back
         self._set_u32s(idxW, u32s)
 
-    def commit(self) -> None:
+    def commit(self, stalled: bool) -> None:
         assert not self.err_flag
+
+        if self._load_begun:
+            self._load_begun = False
+            self._load_ready = True
+        else:
+            self._load_ready = False
+
+        if stalled:
+            return
```

Comment:
Is this early exit needed? Since we don't have stalling stores, I'd be inclined to leave it out.